### PR TITLE
NO-ISSUE add FIO Image and CVO check parameters

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

**Review by nmagnezi — APPROVED** [review]
/lgtm

File: discovery-infra/update_assisted_service_cm.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
     ("INVENTORY_IMAGE", ""),
     ("OCM_BASE_URL", ""),
     ("PUBLIC_CONTAINER_REGISTRIES", "")
```
Comment (irosenzw) [inline discovery-infra/update_assisted_service_cm.py:39]: add `,` at the end

File: discovery-infra/update_assisted_service_cm.py [hunk 2]
```diff
@@ -37,6 +37,8 @@
...
+    ("FIO_PERF_CHECK_IMAGE", "")
```
Comment (irosenzw) [inline discovery-infra/update_assisted_service_cm.py:40]: here too.

Comment (rollandf) [reply discovery-infra/update_assisted_service_cm.py]: done

**Review by nmagnezi — APPROVED** [review]
/lgtm

**Review by nmagnezi — COMMENTED** [review]
/test e2e-metal-single-node-live-iso

**Review by nmagnezi — COMMENTED** [review]
/retest

**Review by gamli75 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -103,6 +103,8 @@ Check the [Install Guide](GUIDE.md) for installation instructions.
 | SSO_URL                     | URL used to fetch JWT tokens for assisted-service authentication                                                                |
 | WORKER_MEMORY               | memory for worker VM, default: 8892MB                                                                                           |
 | PUBLIC_CONTAINER_REGISTRIES | comma-separated list of registries that do not require authentication for pulling assisted installer images                     |
+| FIO_PERF_CHECK_IMAGE        | FIO performance check image to use, will update assisted-service config map with given value                                    |
+| CHECK_CLUSTER_VERSION       | If enable, the controller will wait for CVO to finish                                                                           |
```
Comment (YuviGold) [inline README.md:107]: What value does it need to get "enabled"? Just being set? or some value? If so - please describe it.

Comment (rollandf) [reply README.md]: Fixed

**Review by YuviGold — APPROVED** [review]

**Pull request merged**